gradle/libs.versions.toml

```diff
@@ -69,7 +69,7 @@ coroutines = "1.8.0-RC2"
 
 # KSP
 # https://github.com/google/ksp/releases.
-ksp = "1.9.21-1.0.16"
+ksp = "1.9.22-1.0.16"
```

Comment:
you're using ksp 1.9.22 but haven't bumped the kotlin one
https://github.com/imashnake0/Animite/blob/a1b51fff776e7421fc713df006c897d7be60cba3/gradle/libs.versions.toml#L67